RealmSwift-swift1.2/Schema.swift

```diff
@@ -44,6 +44,16 @@ public final class Schema: Printable {
 
     // MARK: Initializers
 
+    /**
+    Creates a `Schema` that contains only the given `objectTypes`.
+
+    :param: objectTypes An array of `Object` subclass types to include in the schema.
+    */
+    public init(objectTypes: [Object.Type]) {
+        self.rlmSchema = RLMSchema(objectClasses: objectTypes)
```

Comment:
this seems not to work in swift 1.2 -- `fatal error: value failed to bridge from Swift type to a Objective-C type`